arm-servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -2974,7 +2974,7 @@
     "EventHealthEvaluation": {
       "type": "object",
       "allOf": [{"$ref": "#/definitions/HealthEvaluation"}],
-      "x-ms-discriminator-value": "1",
+      "x-ms-discriminator-value": "EventHealthEvaluation",
```

Comment:
Oh awesome. I just wanted to be sure that this has not been done due to any shortcoming from the AutoRest side. Btw, I like the approach of using a readable string v/s a number.